compiler/qsc_parse/src/item/tests.rs

```diff
@@ -975,31 +1011,15 @@ fn doc_without_item() {
         }",
         &expect![[r#"
             Namespace _id_ [0-62] (Ident _id_ [10-11] "A"):
-                Item _id_ [26-62]:
+                Item _id_ [0-0]:
```

Comment:
This `0-0` span seems off - it should match the span of the doc comment (see my comment above about using `s.span(lo)` for how to achieve that)